### PR TITLE
Updated contributing guidelines

### PR DESCRIPTION
**Pull request by PyvesB**
As mentioned in #1516, here's a small pull request to make the acceptance criteria for logos more precise.

I've also added a small section encouraging people to star the repository, please comment if you don't think this is a good idea.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -128,7 +131,7 @@ Function declarations are placed in `lib/`, not directly in `server.js`.
 Logos
 -----
 
-Please minimize checked-in SVG files through [SVGO][]. You can use [svgomg][].
+Please minimize checked-in SVG files through [SVGO][]. You can use [svgomg][]. In general we only accept logos that have a corresponding badge on the homepage (e.g. the Eclipse logo because we support service badges for the Eclipse Marketplace), but we may also approve logos for tools widely used by developers (e.g. our Slack logo).
```
Comment (paulmelnikow) [inline CONTRIBUTING.md]: How about adding something suggesting people not expect this to happen... maybe like, “Don’t count on this. It’s at the discretion of the maintainers.” Or along those lines. My goal is that maintainers feel comfortable saying no.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -67,6 +67,9 @@ don't see it, feel free to [open a new issue][open an issue].
 
 [open an issue]: https://github.com/badges/shields/issues/new
 
+### Starring the repository
+
+Feel free to star the repository. This will help increase the visibility of the project, therefore attracting more users and contributors to Shields!
```
Comment (paulmelnikow) [inline CONTRIBUTING.md]: Great idea to include this! I’m cool with it as is. Another idea: do you think we should expand the call to action to “Spread the word,” and include asking people to share that we ask developers who to use and love shields to give $10.

**Comment by paulmelnikow** [comment]
Thanks for opening this!

File: CONTRIBUTING.md (hunk 1 above)
Comment (PyvesB) [reply CONTRIBUTING.md]: How about now? I've tried to rephrase your suggestion in a slightly more welcoming way, hopefully the meaning is not too diluted!

File: CONTRIBUTING.md (hunk 2 above)
Comment (PyvesB) [reply CONTRIBUTING.md]: Done, how does it look?

Comment (paulmelnikow) [reply CONTRIBUTING.md]: Woot!

File: CONTRIBUTING.md (hunk 1 above)
Comment (paulmelnikow) [reply CONTRIBUTING.md]: I like the way you phrased it. Welcoming is good. I'm just looking out for maintainers here, and wanting people to feel comfortable saying no.

**Pull request merged**